Fabric.Authorization.API/Configuration/DiscoveryServiceSettings.cs

```diff
@@ -3,7 +3,10 @@
     public class DiscoveryServiceSettings
     {
         public string AccessControlToken { get; set; }
+        public string IdentityServiceToken { get; set; }
+        public string AccessControlUIToken { get; set; }
```

Comment:
Do we need this because we need to setup the reply URL?

Reply:
Its used to generate the token with Access control.  The UI makes a call back to OICD.html, which need the current location.  I could get the current location but thought that if we set this somewhere like auth service, then it would be more consistant.

i.e. avoids localhost/<hostname>/127.0.0.1 domain name confusion